packages/integrations/src/jellyfin/jellyfin-integration.ts

```diff
@@ -0,0 +1,57 @@
+import { Jellyfin } from "@jellyfin/sdk";
+import { getSessionApi } from "@jellyfin/sdk/lib/utils/api/session-api";
+
+import { Integration } from "../base/integration";
+import type { StreamSession } from "../interfaces/media-server/session";
+
+const jellyfin = new Jellyfin({
+  clientInfo: {
+    name: "Homarr",
+    version: "0.0.1",
+  },
+  deviceInfo: {
+    name: "Homarr",
+    id: "homarr",
+  },
+});
+
+export class JellyfinIntegration extends Integration {
+  async getCurrentSessionsAsync(): Promise<StreamSession[]> {
+    const apiKey = this.getSecretValue("apiKey");
+    const api = jellyfin.createApi(this.integration.url, apiKey);
+    const sessionApi = getSessionApi(api);
+    const sessions = await sessionApi.getSessions();
+
+    if (sessions.status !== 200) {
+      throw new Error(
+        `Jellyfin server ${this.integration.url} returned a non successful status code: ${sessions.status}`,
+      );
+    }
+
+    return sessions.data.map((sessionInfo): StreamSession => {
+      let nowPlaying: StreamSession["currentlyPlaying"] | null = null;
+
+      if (sessionInfo.NowPlayingItem) {
+        nowPlaying = {
+          type: "tv",
+          name: sessionInfo.NowPlayingItem.Name ?? "",
+          seasonName: sessionInfo.NowPlayingItem.SeasonName ?? "",
+          episodeName: sessionInfo.NowPlayingItem.EpisodeTitle,
+          albumName: sessionInfo.NowPlayingItem.Album ?? "",
+          episodeCount: sessionInfo.NowPlayingItem.EpisodeCount,
+        };
+      }
+
+      return {
+        sessionId: `${sessionInfo.Id}`,
+        sessionName: `${sessionInfo.Client} (${sessionInfo.DeviceName})`,
+        user: {
+          profilePictureUrl: null,
+          userId: `${sessionInfo.UserId}`,
```

Comment:
Not sure but this seems completly unnecessary

Reply:
You mean this? All properties are nullable, even if the API doesn't seem to return null (unclear?)
![image](https://github.com/homarr-labs/homarr/assets/30572287/c89676af-6b83-440e-b181-3d7fcb370cbe)

Reply:
![image](https://github.com/homarr-labs/homarr/assets/63781622/93d39c98-1e89-4e6e-8bca-bc435dde62df)
I think we should fallback to an empty string then